### PR TITLE
Update binding flags, and pick the correct httprequest class

### PR DESCRIPTION
**Pull request by Tatsujinichi**
For Issue
BindingFlags and Invoke Member Unity 2017.1 #61

not tested on earlier versions of unity, 
works on 2017.1.0f3

### CODE REVIEW

**Comment by Tatsujinichi** [comment]
@nullorvoid can you review this

**Comment by nullorvoid** [comment]
Logic looks fine, no problem with this, but it will need to be tested on 5.5.3 before I will merge it. Would you mind doing this? I can do this, but at the earliest will be Tuesday next week

File: Mage/Mage.cs [hunk 1]
```diff
@@ -131,8 +132,8 @@ public void SetupModules(List<string> moduleNames, Action<Exception> cb)
 					{
 						if (moduleName == t.Name)
 						{
-							BindingFlags staticProperty = BindingFlags.Static | BindingFlags.GetProperty;
-							BindingFlags publicMethod = BindingFlags.Public | BindingFlags.InvokeMethod;
+							BindingFlags staticProperty = BindingFlags.InvokeMethod | BindingFlags.GetField | BindingFlags.GetProperty;
```
Comment (AlmirKadric) [inline Mage/Mage.cs:135]: Provided these flag changes work on all devices, LGTM 👍

**Comment by AlmirKadric** [comment]
@Tatsujinichi why some of the lines showing as changed although they're not
Line endings?

Other than above comments LGMT 👍

**Comment by Tatsujinichi** [comment]
I assume my editor was converting spaces to tabs, but I could be wrong.

**Comment by AlmirKadric** [comment]
@Tatsujinichi 
Both lines in the diff have tabs, its why i'm assuming line endings. Not really a big deal, more a nuisance and it could even be the line endings are wrong in the original code. If you could just check what it is so we know

**Comment by Tatsujinichi** [comment]
Yes, it was line endings.
Before:  CRFL  ( 0D 0A )
before
After: LF ( 0A )
after

I believe the LF is the correct ending.

**Comment by AlmirKadric** [comment]
👍

**Comment by Tatsujinichi** [comment]
Actually I am not sure the unity version of httprequest is working at all.  I've switched back to the .net version.

**Comment by nullorvoid** [comment]
We should probably move it from WWW to UnityWebRequest which is much newer and probably better supported.

**Comment by Tatsujinichi** [comment]
closing this as https://github.com/mage/mage-sdk-unity/pull/66 should accomplish the same thing

**Pull request closed without merging**